api/Program.cs

```diff
@@ -4,6 +4,7 @@
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.IdentityModel.Tokens;
 using Microsoft.OpenApi.Models;
+using Model;
```

Comment:
Kan je deze ook weghalen. Bij Order.cs is daar nog een namespace model die weg moet. :)